packages/stark-ui/src/modules/minimap/components/minimap.component.spec.ts

```diff
@@ -12,7 +12,7 @@ import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing
 import { Component, EventEmitter, NO_ERRORS_SCHEMA, ViewChild } from "@angular/core";
 import { MatCheckboxModule } from "@angular/material/checkbox";
 import { MatTooltipModule } from "@angular/material/tooltip";
-import { ItemVisibility } from "@nationalbankbelgium/stark-ui";
+import { ItemVisibility } from "../../minimap";
```

Comment:
The import should be refined like this: `../item-visibility.intf`, otherwise is redundant cause it will import everything again from "minimap" and that could case new "circular dependecies" later on 😉